ui/src/main/webapp/src/app/components/rules-modal.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal fade" id="rulesModal" tabindex="-1" role="dialog" aria-labelledby="Rules List" aria-hidden="true">
-    <div class="modal-dialog modal-lg">
+    <div class="modal-dialog modal-lg" (mouseover)="prettyPrint()">
```

Comment:
(17:26:33) ozizka: I wasn't able to find a way to call a function when the modal is shown.
(17:26:41) ozizka: So there's a timeout and mouseover.